### ISSUE TITLE
Question about the train/val/test sets

### ISSUE THREAD

**Issue by MenglaiWang** [zhaoweicai/mscnn#17]
Thanks for sharing your code! 
I have read some of source code. I am confused about your split of train/val/test sets.
In the Caltech  train_val.prototxt ,  [training set](https://github.com/zhaoweicai/mscnn/blob/master/examples/caltech/mscnn-7s-720/trainval_1st.prototxt#L23) is set00-05, the validation set is [this](https://github.com/zhaoweicai/mscnn/blob/master/examples/caltech/mscnn-7s-720/trainval_1st.prototxt#L80)
![image](https://cloud.githubusercontent.com/assets/11674319/19415870/2ad9b44c-93b1-11e6-922c-dde9477b5485.png)
As shown in the figure, in the  mscnn_window_file_caltech_test.txt , the validation images are from set06-10. 
So your testing set is the same as your validation set?? 
I wonder whether you have upload the wrong validation sets or I have a misunderstanding about Caltech pedestrian datasets.
Hope for your reply.

**Comment by zhaoweicai** [comment]
Since the ground truth for Caltech testing set is available, we do not split the training set into train/val sets here. This follows other works on VOC2007 dataset. You can split the train/val sets by yourself.
